arangodb-net-standard/AnalyzerApi/Models/AnalyzerProperties.cs

```diff
@@ -107,5 +116,256 @@ public class AnalyzerProperties
+        /// the batch size for reading data from the query. In general, 
+        /// a single token is expected to be returned. However, if the 
+        /// query is expected to return many results, then increasing 
+        /// <see cref="BatchSize"/> trades memory for performance.
+        /// </summary>
+        public int? BatchSize { get; set; }
+
+        /// <summary>
+        /// Memory limit for query execution in bytes. 
+        /// (default is 1048576 = 1Mb) 
+        /// Maximum is 33554432U (32Mb)
+        /// </summary>
+        public int? MemoryLimit { get; set; }
+
+        /// <summary>
+        /// The Data type of the returned tokens. If the indicated 
+        /// type does not match the actual type then an implicit 
+        /// type conversion is applied. For possible values, see
+        /// <see cref="ReturnTypeBool"/>, <see cref="ReturnTypeNumber"/>
+        /// and <see cref="ReturnTypeString"/>
+        /// </summary>
+        public string ReturnType { get; set; }
+
+        /// <summary>
+        /// The delimiting character(s)
+        /// </summary>
+        public string Delimiter { get; set; }
+
+        /// <summary>
+        /// Index all GeoJSON geometry types (Point, Polygon etc.)
+        /// (default).
+        /// </summary>
+        public const string TypeShape = "shape";
+
+        /// <summary>
+        /// Compute and only index the centroid of the input 
+        /// geometry
+        /// </summary>
+        public const string TypeCentroid = "centroid";
+
+        /// <summary>
+        /// Only index GeoJSON objects of type Point, ignore all 
+        /// other geometry types
+        /// </summary>
+        public const string TypePoint = "point";
+
+        /// <summary>
+        /// Determines the type of indexing to use.
+        /// Possible values are <see cref="TypeCentroid"/>, 
+        /// <see cref="TypeShape"/> and <see cref="TypePoint"/>
+        /// </summary>
+        public string Type { get; set; }
+
+        /// <summary>
+        /// Options for fine-tuning geo queries. 
+        /// These options should generally remain unchanged.
+        /// </summary>
+        public GeoJSONAnalyzerOptions Options { get; set; }
+
+        /// <summary>
+        /// A list of strings that describes the attribute path 
+        /// of the latitude value relative to the field for 
+        /// which the Analyzer is defined in the View.
+        /// </summary>
+        public List<string> Latitude { get; set; }
+
+        /// <summary>
+        /// A list of  strings that describes the attribute path
+        /// of the longitude value relative to the field for 
+        /// which the Analyzer is defined in the View.
+        /// </summary>
+        public List<string> Longitude { get; set; }
+
+        /// <summary>
+        /// Convert to all lower-case characters
+        /// </summary>
+        public const string CaseHandlingLower = "lower";
```

Comment:
Would have been good to group the const together. I don't understand how they are grouped right now. But that doesn't really matter as we are going to review this in #450